### PR TITLE
bump mender-convert and raspios image

### PR DESCRIPTION
**Pull request by oleorhagen**
Note: Not checked for the img availability yet, but should be good.

Will test before merging.

### CODE REVIEW

**Review by kacf — APPROVED** [review]
Be sure to test this. We've had some reports on Mender Hub about non-working conversions.

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ Download the Raspberry Pi OS image ready for Mender:
   * Download link: [Raspberry Pi 4 Model B][raspios-buster-lite-raspberrypi4-mender.img.xz]
 
 <!--AUTOVERSION: "mender-convert-%.img.xz"/mender-convert -->
-[raspios-buster-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2021-01-11-raspios-buster-armhf-lite/arm/2021-01-11-raspios-buster-armhf-lite-raspberrypi3-mender-convert-master.img.xz
-[raspios-buster-lite-raspberrypi4-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2021-01-11-raspios-buster-armhf-lite/arm/2021-01-11-raspios-buster-armhf-lite-raspberrypi4-mender-convert-master.img.xz
+[raspios-bullseye-armhf-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2021-10-30-raspios-bullseye-armhf-lite/arm/2021-10-30-raspios-bullseye-armhf-lite-raspberrypi3-mender-convert-master.img.xz
```
Comment (lluiscampos) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: This is not correct, the reference above is not updated so the link won't be rendered correctly. Line 33 still reads `raspios-buster-lite-raspberrypi3-mender.img.xz`.

Also, although harmless, we don't need to add the suffix `-armhf`.

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md (hunk 1 above)
Comment (oleorhagen) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: https://github.com/mendersoftware/mender-docs/pull/1626

fixed here for 3.2.x. Needs cherry for master

Comment (lluiscampos) [reply 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:37]: > Needs cherry for master

This PR goes to master so we are good.

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**